src/Test/TestAIBehavior.java

```diff
@@ -293,13 +294,14 @@ private boolean testTankWadeThroughInfs(AIMaker ai)
       addUnit(testMap, testCo1, "Infantry", new XYCoord(xx, 0));
     Unit nmeArty = addUnit(testMap, testCo2, "Artillery", new XYCoord(9, 0)); // Target artillery
     Unit inf = addUnit(testMap, testCo1, "Md Tank", new XYCoord(1, 0)); // Create tank last just to make the AI compensate.
+    turn(testGame);
 
     // The inf will want to cap the enemy HQ. Make sure the tanks get out of the way.
     boolean testPassed = true;
     int turnLimit = 4; // This should be enough time to cap the enemy HQ.
     for( int tt = 0; tt < turnLimit; ++tt )
     {
-      testCo1.army.initTurn(testMap);
+      turn(testGame, 2);
```

Comment:
`testGame.armies.length`? I guess that might not track correctly if COs are defeated; I don't remember if we remove them from the list.

Reply:
We don't.
I dislike magic even less in test code, though.
Tests are supposed to be at least a little brittle.